### ISSUE TITLE
Datatable column showClearButton properties not working

### ISSUE THREAD

**Issue by ahbapAldirmaz** [primefaces/primevue#1606]
```
<Column
				field="quoteAsset"
				:showFilterMatchModes="false"
				:showFilterMenu="true"
				:header="$t('parity')"
				style="max-width: 100px"
				:sortable="false"
				:showClearButton="false"
				:showApplyButton="false"
			>
```
:showClearButton="false"

Clear Button is always showing in the FilterMenu at the Datatable. 

package.json --> "primevue": "^3.7.1",

can you help me?

**Comment by nclemeur** [comment]
Maybe you can check #1532, and see if if updating to the latest solve your issue?

**Comment by tugcekucukoglu** [comment]
> Maybe you can check #1532, and see if if updating to the latest solve your issue?

Fixed for 3.7.2
